src/BlazorWebFormsComponents/ListView.razor

```diff
@@ -30,31 +21,47 @@ else if (Items == null || !Items.Any())
 else
 {
 
-	if (OnDataBinding.HasDelegate) OnDataBinding.InvokeAsync(EventArgs.Empty);
-
-	@foreach (var item in Items)
+	@if (LayoutTemplate is null)
 	{
+		LayoutTemplate = context => builder => builder.AddContent(1, context);
+	}
 
+	@LayoutTemplate(
+			@: @if (Items != null)
```

Comment:
This is where the magic happens - by switching context to HTML then starting a code block immediately, we can create a `RenderFragment` inline - effectively everything in the if statement becomes a `RenderFragment` that we can then inject into the `LayoutTemplate` as its `Context`